portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,24 +79,60 @@ public void doGet (HttpServletRequest request, HttpServletResponse response) thr
   @Override
    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
+     String[] nameErrorArr = new String[]{"invalid name input"};
+     String[] textErrorArr = new String[]{"invalid text input"};
+     String username = request.getParameter("name-input");
      String text = request.getParameter("text-input");
 
+    
+    if(username.isEmpty()){
+            Gson gson = new Gson();
+            response.setContentType("application/json");
+            response.getWriter().println(gson.toJson(nameErrorArr));
```

Comment:
it looks like nameErrorArr isn't used anywhere.  You can just do

response.getWriter().println(gson.toJson(new String[]{"invalid name input"}));